README.rst

```diff
@@ -85,6 +131,34 @@ Setting Up Your Users
 
 For convenience, you can keep your normal user logged in on Chrome and your superuser logged in on Firefox (or similar), so that you can see how the site behaves for both kinds of users.
 
+If the command prompts for username only and after entering if it goes to error
+do make sure that you have done the following 
+
+Note: Make sure that you have created a database named `care` with privileges set for the user `postgres`
+
+In the virtualenv shell type the following commands also.
+
+`export DATABASE_URL=postgres://postgres:<password>@127.0.0.1:5432/care`
+
+`export TEST_POSTGIS_URL="postgis://postgres:<password>@127.0.0.1:5432/care"`
+
+`python manage.py migrate`
+
+and after you make sure everything is ok
+
+run this command again
+
+`python manage.py createsuperuser`
+
+This will now prompt for the following details - Ignore any warnings.
+
+- username: give the username here
+- usertype:Give the value `10` [5 for doctor, 10 for hospital staff/hospital administrator, 15 for patient, 20 for volunteer]
+- gender: 1 for male, 2 for female, 3 for other
```

Comment:
non-binary is the term not "others"

Reply:
where should I use non-binary instead of others? In the gender: expansion?